{"timestamp": "2025-01-01 14:57:05", "username": "System", "logger_name": "__main__", "log_level": "INFO", "message": "Starting the application."}
{"timestamp": "2025-01-01 14:57:05", "username": "System", "logger_name": "__main__", "log_level": "INFO", "message": "Table 'users' initialized successfully."}
{"timestamp": "2025-01-01 14:57:05", "username": "System", "logger_name": "__main__", "log_level": "INFO", "message": "Admin user already exists."}
{"timestamp": "2025-01-01 14:57:05", "username": "System", "logger_name": "__main__", "log_level": "INFO", "message": "Services initialized successfully."}
{"timestamp": "2025-01-01 14:57:08", "username": "System", "logger_name": "__main__", "log_level": "INFO", "message": "[AUDIT] 2025-01-01 14:57:08.644257 - User: jmp637, Action: Login failed for user 'jmp637'"}
{"timestamp": "2025-01-01 14:57:20", "username": "jmp637", "logger_name": "__main__", "log_level": "INFO", "message": "User jmp637 logged in."}
{"timestamp": "2025-01-01 14:57:21", "username": "jmp637", "logger_name": "__main__", "log_level": "ERROR", "message": "An unexpected error occurred: 'MainWindow' object has no attribute 'module_manager'"}
{"timestamp": "2025-01-01 14:57:21", "username": "jmp637", "logger_name": "__main__", "log_level": "DEBUG", "message": "Traceback (most recent call last):\n  File \"C:\\Users\\Lenovo\\OneDrive\\Documents\\GitHub\\Shiva\\main.py\", line 63, in main\n    main_window = MainWindow(user_model, logger=logger)\n  File \"C:\\Users\\Lenovo\\OneDrive\\Documents\\GitHub\\Shiva\\ui\\main_window.py\", line 70, in __init__\n    self.create_menu_bar()\n  File \"C:\\Users\\Lenovo\\OneDrive\\Documents\\GitHub\\Shiva\\ui\\main_window.py\", line 100, in create_menu_bar\n    self.module_manager.load_module_options(module_dir)\nAttributeError: 'MainWindow' object has no attribute 'module_manager'\n"}
{"timestamp": "2025-01-01 14:57:23", "username": "jmp637", "logger_name": "__main__", "log_level": "INFO", "message": "Database connection closed."}
